src/pudl/transform/params/ferc1.py

```diff
@@ -1876,47 +1954,143 @@
                 "required_valid_cols": ["plant_name_ferc1"],
             },
         ],
-        # "spot_fix": [
-        #     {
-        #         "record_id_ferc1": "f1_steam_1999_12_72_0_1",
-        #         "fixes": {"plant_name_ferc1": "clifty creek"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_2010_12_306_0_1",
-        #         "fixes": {"plant_name_ferc1": "harrison county"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_1997_12_230_0_1",
-        #         "fixes": {"plant_name_ferc1": "hermiston generating"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_1998_12_64_0_1",
-        #         "fixes": {"plant_name_ferc1": "hardee power station"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_2015_12_276_0_1",
-        #         "fixes": {"plant_name_ferc1": "state line"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_2014_12_276_0_1",
-        #         "fixes": {"plant_name_ferc1": "state line"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_2003_12_62_2_3",
-        #         "fixes": {"plant_name_ferc1": "pea ridge"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_2003_12_62_2_2",
-        #         "fixes": {"plant_name_ferc1": "smith"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_2000_12_204_0_1",
-        #         "fixes": {"plant_name_ferc1": "seabrook"},
-        #     },
-        #     {
-        #         "record_id_ferc1": "f1_steam_2001_12_204_0_1",
-        #         "fixes": {"plant_name_ferc1": "seabrook"},
-        #     },
-        # ],
+    },
+    "plants_hydro_ferc1": {
+        "rename_columns_ferc1": {
+            "dbf": {
+                "columns": {
```

Comment:
@zaneselvans  and I and maybe even also ben went back and forth on this a fair amount in the og transformer pr iirc. I think i agree with you generally... but I think we went with this setup bc this is the format that we actually need when we run `df.rename`. so we would need to compile it in this format anyway.

There are columns that don't exist in both xbrl and dbf and it would be clearer which ones don't overlap. or when there are instances in which the units or some other thing don't overlap and thus need to be treated before these are concatenated.

Reply:
The two data sources don't have exactly the same set of columns, and I found the combined data structure harder to read and work with. You have to construct the dictionaries before using them in `df.rename` rather than being able to apply them directly as they are stored.

Reply:
Ya that makes sense. Just a design difference. Don't think we need to change anything at this point.